### DISCUSSION TITLE
Combine two datasets

### DISCUSSION THREAD

**Discussion by anthonysirico** [pyg-team/pytorch_geometric#5759] [Q&A] [answered]
How do you combine two datasets. The way I did was:
```
Big_set = set1 + set2
```
But that does not carry over all the data. Should I use:
```
Big_set = set1.copy() + set2.copy()
```
or is there another way?

**Comment by rusty1s** [comment, marked as answer]
This is directly supported within PyTorch using `ConcatDataset`. Please take a look [here](https://pytorch.org/docs/stable/data.html#torch.utils.data.ConcatDataset).
